crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/bandwidth.rs

```diff
@@ -245,8 +257,12 @@ mod tests {
 
     #[tokio::test(flavor = "current_thread", start_paused = true)]
     async fn test_limiter() {
+        observability::test_run().ok();
         let clock = governor::clock::FakeRelativeClock::default();
-        let bandwidth = BandwidthThrottle::test(0.1, 0.1, clock.clone());
+        // max * 2 * 8 = 0.1 * 1_000_000 * burst_ratio => burst_ratio = max * 2 * 8 / 0.1 / 1_000_000
+        let burst_ratio = MAX_SEND_BUF_BYTES as f64 * 2.0 * 8.0 / 1_000_000.0 / 0.1;
```

Comment:
Where is `MAX_SEND_BUF_BYTES` defined? If we're not using it anymore in the real code, should it be moved into this test-only block?

Reply:
It is used elsewhere too, like here https://github.com/holochain/holochain/blob/9ee4533108e408aabce1f489aa23dfcee581ee4a/crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/bandwidth.rs#L96

Reply:
and here https://github.com/holochain/holochain/blob/90ea430b3286bfa4db4c746c3492429a323951e1/crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/ops.rs#L308

Reply:
o_O - isn't bandwidth.rs:96 deleted / replaced by your new tuning_param?? - Should ops.rs:308 be also?

Reply:
:facepalm: I was on the wrong branch when I looked at those. Yes, the first one is deleted. But the second one is not, and is still important. The deletion of the first instance just makes the burst limit explicit instead of being dependent on the max chunk size.